From 384d1ed0af7922804d3f6b83d94490b4d69dc963 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Fri, 13 Jan 2012 15:45:43 -0500
Subject: [PATCH] build: Note that we don't support srcdir != builddir

---
 configure.in |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/configure.in b/configure.in
index 7af2e8d..75853e3 100644
--- a/configure.in
+++ b/configure.in
@@ -19,6 +19,9 @@ AC_CONFIG_MACRO_DIR([m4])
 AC_CONFIG_HEADER(include/config.h)
 AC_PREFIX_DEFAULT(/usr)
 
+dnl http://people.gnome.org/~walters/docs/build-api.txt
++echo \#buildapi-variable-no-builddir >/dev/null
+
 AC_PROG_LIBTOOL
 
 AC_ARG_ENABLE(shared,
-- 
1.7.6.4

