From 79f8f5db8b484b51ee8e18ad464fb789dd4cd3e5 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Fri, 20 Jan 2012 17:08:52 -0500
Subject: [PATCH] build: Note we don't support srcdir != builddir

---
 configure.ac |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/configure.ac b/configure.ac
index 8dcd441..3105995 100644
--- a/configure.ac
+++ b/configure.ac
@@ -11,6 +11,8 @@ AC_CONFIG_HEADERS([config.h])
 AM_INIT_AUTOMAKE([1.11 -Wno-portability no-dist-gzip dist-xz])
 AM_SILENT_RULES([yes])
 
+echo \#buildapi-variable-no-builddir >/dev/null
+
 # Check for programs
 AC_PROG_CC
 AM_PROG_VALAC([0.11.7])
-- 
1.7.6.5

