From 23b2d7561fbc609ab2a00c2a261a203587fb802b Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Tue, 17 Jan 2012 17:12:07 -0500
Subject: [PATCH] build: Note we don't support srcdir != builddir

---
 configure.ac |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/configure.ac b/configure.ac
index 0e3b20e..c811a58 100644
--- a/configure.ac
+++ b/configure.ac
@@ -14,6 +14,9 @@ AG_GST_INIT
 dnl initialize automake
 AM_INIT_AUTOMAKE([-Wno-portability 1.10])
 
+dnl http://people.gnome.org/~walters/docs/build-api.txt
+echo \#buildapi-variable-no-builddir >/dev/null
+
 dnl define PACKAGE_VERSION_* variables
 AS_VERSION
 
-- 
1.7.6.5

