From 2d4163a099a3c44721125105e0f8f81bfebb7e53 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Thu, 19 Jan 2012 18:45:55 -0500
Subject: [PATCH] Make testarraylist compile again

---
 tests/testarraylist.vala |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/tests/testarraylist.vala b/tests/testarraylist.vala
index b13ec30..42d02a5 100644
--- a/tests/testarraylist.vala
+++ b/tests/testarraylist.vala
@@ -131,7 +131,7 @@ public class ArrayListTests : BidirListTests {
 		assert (double_list.add (1.5d));
 		assert (double_list.add (2.0d));
 
-		double[] double_array = double_list.to_array ();
+		double?[] double_array = double_list.to_array ();
 		index = 0;
 		foreach (double element in double_list) {
 			assert (element == double_array[index++]);
-- 
1.7.6.5

