From fa0949cfb146f4d412e035923340ea4265dac4db Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Fri, 20 Jan 2012 13:11:31 -0500
Subject: [PATCH] Install telepathy-glib.vapi to vala-0.16 directory

---
 vala/Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/vala/Makefile.am b/vala/Makefile.am
index faefefa..2d6dbb7 100644
--- a/vala/Makefile.am
+++ b/vala/Makefile.am
@@ -1,4 +1,4 @@
-vapidir = $(datarootdir)/vala/vapi
+vapidir = $(datarootdir)/vala-0.16/vapi
 vapi_DATA = \
 	telepathy-glib.vapi \
 	$(NULL)
-- 
1.7.6.5

