From c80f8fbda32f8bd3d2535686775493de8d0f60d1 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Fri, 20 Jan 2012 15:59:01 -0500
Subject: [PATCH] configure: Note we don't support srcdir != builddir

---
 configure.ac |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/configure.ac b/configure.ac
index a29efdd..69cb0b2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -45,6 +45,9 @@ AC_CONFIG_MACRO_DIR([m4])
 AM_INIT_AUTOMAKE([1.11 -Wno-portability])
 AM_CONFIG_HEADER(config.h)
 
+dnl vapigen fails to locate .metadata file when srcdir != builddir
+echo \#buildapi-variable-no-builddir >/dev/null
+
 AM_SILENT_RULES
 
 dnl check for tools
-- 
1.7.6.5

