From f3c2cc5eab452e5eb091a6687460cad033b82041 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Sat, 14 Jan 2012 16:14:18 -0500
Subject: [PATCH] build: Note that we don't support srcdir != builddir

---
 configure.ac |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/configure.ac b/configure.ac
index 1643aec..a794bfc 100644
--- a/configure.ac
+++ b/configure.ac
@@ -4,6 +4,9 @@ AM_INIT_AUTOMAKE([1.9 foreign dist-bzip2 no-dist-gzip])
 AC_CONFIG_SRCDIR([src])
 AC_CONFIG_HEADERS([config.h])
 
+# http://people.gnome.org/~walters/docs/build-api.txt
+echo \#buildapi-variable-no-builddir >/dev/null
+
 AM_MAINTAINER_MODE
 
 AC_PROG_CC
-- 
1.7.6.4

